src/crc32.cc

```diff
@@ -0,0 +1,84 @@
+#include "crc32.hh"
```

Comment:
we already have hash_data which uses murmur3 and should be fast enough. I dont think we really need to add a crc32 implementation.

Reply:
Side question, is `murmur3` that much better than a plain old CRC32? It seems to be using substantially more cycles, maybe be it's not the best algorithm to hash entire files of arbitrary length? Using it for key hashes is sound enough (because of collisions), I wonder about files (whose contents are generally plain text).